qemu/README.soo

```diff
@@ -22,13 +22,7 @@ QEMU Configuration and compilation
 
 Configure options:
 
-AArch32 version:
-
-./configure --target-list=arm-softmmu --disable-attr --disable-werror --disable-docs
-
-AArch64 version:
-
-./configure --target-list=aarch64-softmmu --disable-attr --disable-werror --disable-docs
+./configure --target-list=arm-softmmu,aarch64-softmmu --disable-attr --disable-werror --disable-docs
```

Comment:
Why removing the qemu 32bit reference ?

Reply:
arm_softmmu: 32-bit
aarch64-softmmu: 64-bit